tests/impact-calculations/impact.gui.ts

```diff
@@ -30,4 +34,35 @@ test.describe("Impact page", () => {
       await page.getByTestId("number-persons-out-poverty").innerText()
     ).toEqual("3");
   });
+
+  test("should simulate lifting people out of poverty", async ({
+    page,
+    gui,
+  }) => {
+    // click on simulateBuyGlo
+    await page.waitForTimeout(3000);
```

Comment:
I'd avoid using `waitForTImeout` at all cost.

Probably the `waitForSelector` is the correct wait so waits aren't constant, but waits for specific element to load.